lang/src/org/partiql/lang/eval/builtins/NullIfExprFunction.kt

```diff
@@ -1,40 +0,0 @@
-/*
```

Comment:
`NULLIF` and `COALESCE` were modeled as nodes part of the `PartiqlAst` internally rather than `ExprFunction`s.